pennylane/ops/qubit/observables.py

```diff
@@ -349,31 +350,107 @@ class Projector(Observable):
     * Gradient recipe: None
 
     Args:
-        basis_state (tensor-like): binary input of shape ``(n, )``
-        wires (Iterable): wires that the projector acts on
+        state (tensor-like): input of shape ``(n, )``.
```

Comment:
```suggestion
        state (tensor-like): input state
```
Note that the shape changes based on the type of input. Maybe we mention the shape in `basis_representation`.

Reply:
Oh is `basis_representation` now removed? Are you planning to do this based on shape?